ffi/diplomat/src/properties_maps.rs

```diff
@@ -2,28 +2,19 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-struct ErasedCodePointMap16MarkerV1;
-
-use icu_provider::DataMarker;
-impl DataMarker for ErasedCodePointMap16MarkerV1 {
-    type Yokeable = <icu_properties::provider::ScriptV1Marker as DataMarker>::Yokeable;
-}
-
 #[diplomat::bridge]
 pub mod ffi {
-    use super::ErasedCodePointMap16MarkerV1;
     use crate::provider::ffi::ICU4XDataProvider;
     use alloc::boxed::Box;
-    use icu_properties::maps;
-    use icu_provider::prelude::DataPayload;
+    use icu_properties::{maps, Script};
 
     use crate::errors::ffi::ICU4XError;
     use diplomat_runtime::DiplomatResult;
 
     #[diplomat::opaque]
     /// An ICU4X Unicode Set Property object, capable of querying whether a code point is contained in a set based on a Unicode property. For properties whose values fit into 16 bits.
     #[diplomat::rust_link(icu_properties, Mod)]
-    pub struct ICU4XCodePointMapData16(DataPayload<ErasedCodePointMap16MarkerV1>);
+    pub struct ICU4XCodePointMapData16(maps::CodePointMapData<Script>);
```

Comment:
Issue: I want this to support any 16-bit value, not just `Script`, so that over FFI we have only `ICU4XCodePointMapData16` and `ICU4XCodePointMapData8` (and 32 if needed). Can you do this with map_project or similar?

I now realize that this was probably retaining the `Script` even before, so this may be a pre-existing issue that we need to fix but can be done in a follow-up

Reply:
Yes, this was a preexisting issue, I wasn't sure what to do here. We'll need to rethink this FFI, I'm trying not to change that right now though.